single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java

```diff
@@ -899,6 +898,67 @@ private void drawFeederInfo(String prefixId, String feederNodeId, List<Point> po
         root.appendChild(g);
     }
 
+    protected void insertBusInfo(String prefixId,
+                                 Point vlShift,
+                                 double shiftX,
+                                 Element root,
+                                 BusNode busNode,
+                                 GraphMetadata metadata,
+                                 DiagramLabelProvider initProvider,
+                                 DiagramStyleProvider styleProvider) {
+        Optional<BusInfo> busInfo = initProvider.getBusInfo(busNode);
+        busInfo.ifPresent(info -> {
+            drawBusInfo(prefixId, busNode, vlShift, shiftX, root, busInfo.get(), styleProvider, metadata);
+            addInfoComponentMetadata(metadata, busInfo.get().getComponentType(), false);
+        });
+    }
+
+    private void drawBusInfo(String prefixId,
+                             BusNode busNode,
+                             Point vlShift,
+                             double shiftX,
+                             Element root,
+                             BusInfo busInfo,
+                             DiagramStyleProvider styleProvider,
+                             GraphMetadata metadata) {
+        Element g = root.getOwnerDocument().createElement(GROUP);
+        // Position
```

Comment:
I find it more readable when there's an empty line before the comment, would you agree?